src/components/Timeline.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import classNames from "classnames";
+
+const defaultProps = {

```

Comment:
don't create a separate object for default props, write them exactly in Timeline.defaultProps = {}